src/DIRAC/Core/DISET/ThreadConfig.py

```diff
@@ -1,5 +1,6 @@
-import threading
 import functools
+import threading
```

Comment:
Why? If we want to sort imports it shouldbe done in one MR like all of the other bulk refactorings.